### DISCUSSION TITLE
Shared blueprints

### DISCUSSION THREAD

**Discussion by david-lobo** [statamic/cms#5964] [Q&A] [answered]
Hi, I am new to Statamic and have a question.  

I have a collection portfolio that uses the portfolio blueprint.  I want to create multiple portfolio collections (design portfolio, services portfolio etc) but each should share the same fields.

Is there a way to assign multiple collections to use the same blueprint? (e.g. design portfolio, services portfolio should use the same portfolio blueprint).

What i tried so far is editing the content/collections/design_portfolio.yaml file to assign multiple blueprints but it doesn't work

blueprints:
  - design_portfolio
  - portfolio

**Comment by afonic** [comment, marked as answer]
You can't but you can create a Fieldset that contains the fields you need and then link it to all these Collection's blueprints.

**Reply by david-lobo** [reply]
Thanks for explaining, will give it a try
